go/oasis-node/cmd/debug/txsource/workload/queries.go

```diff
@@ -239,9 +240,11 @@ func (q *queries) doConsensusQueries(ctx context.Context, rng *rand.Rand, height
 			"txs_with_results", txsWithRes,
 			"height", height,
 			"err", err,
+			"status", cmnGrpc.GetErrorStatus(err),
 		)
-		if status := cmnGrpc.GetErrorStatus(err); status != nil {
-			if status.Err() == io.ErrUnexpectedEOF {
+		if st := cmnGrpc.GetErrorStatus(err); st != nil {
+			s, _ := status.FromError(io.ErrUnexpectedEOF)
+			if st.Err().Error() == s.Err().Error() {
```

Comment:
Ugh this seems awkward.

Reply:
Yeah, unfortunately i didn't find a nicer way of doing this (the alternative would be directly matching on string - well which is kinda close to what this is doing, but at least here there's no hardcoded strings here :P)